pkg/config/log.go

```diff
@@ -102,8 +102,11 @@ func SetupLogger(loggerName LoggerName, logLevel, logFile, syslogURI string, sys
 // if a non empty logFile is provided, it will also log to the file
 // a non empty syslogURI will enable syslog, and format them following RFC 5424 if specified
 // you can also specify to log to the console and in JSON format
-func SetupJMXLogger(loggerName LoggerName, logLevel, logFile, syslogURI string, syslogRFC, logToConsole, jsonFormat bool) error {
-	seelogLogLevel, err := validateLogLevel(logLevel)
+func SetupJMXLogger(loggerName LoggerName, logFile, syslogURI string, syslogRFC, logToConsole, jsonFormat bool) error {
+	// The JMX logger always logs at level "info", because JMXFetch does its
+	// own level filtering on and provides all messages to seelog at the info
+	// or error levels, via log.JMXInfo and log.JMXError.
+	seelogLogLevel, err := validateLogLevel("info")
```

Comment:
I could be wrong but I think using `info`/`trace`/`debug` worked fine and it was just the higher ones (`warn`/ `error`) that had issues. With this change, we would no longer be able to use debug/trace levels which would be a problem for troubleshooting/debugging if that's the case.

Reply:
Update from offline sync: The jmxfetch configuration which I was referring to is different from the agent's log level of processing of received messages from jmxfetch so my earlier comment shouldn't apply.